server/src/user/user.controller.spec.ts

```diff
@@ -2,20 +2,22 @@ import { when } from 'jest-when';
 import { plainToInstance } from 'class-transformer';
 import { Test, TestingModule } from '@nestjs/testing';
 
+import { SessionInfo } from 'src/common/d';
 import { UserService } from './user.service';
 import { UserController } from './user.controller';
-import { CreateUserRequestDto } from './dto/create-user.dto';
+import { CreateUserRequest } from './dto/create-user.dto';
+import { CREATE_USER } from './../test/stub';
+import { LikeService } from './../like/like.service';
```

Comment:
```suggestion
import { CREATE_USER } from '../test/stub';
import { LikeService } from '../like/like.service';
```

이거 vscode가 좀 이상하게 불러오기를 하더라구요
맨앞의 `./` 은 불필요한 듯합니다 (별로 중요한 부분은 아니지만..)